docs-src/httproute.md

```diff
@@ -0,0 +1,303 @@
+      - path:
+          type: Prefix
+          value: /
+      forwardTo:
+      - port: 8080
+        backendRef:
+          name: my-custom-resource
+          group: acme.io
+          kind: CustomBackend
+```
+
+The following example uses the `weight` field to forward HTTP requests with Host header "my.filter.com" equally across
+service "my-filter-svc1" and service "my-filter-svc2", i.e. traffic splitting:
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-filter-1
+spec:
+  hostnames:
+    - my.filter.com
+  rules:
+    - matches:
+      - path:
+          type: Prefix
+          value: /
+      forwardTo:
+      - serviceName: my-filter-svc1
+        port: 80
+        weight: 50
+      - serviceName: my-filter-svc2
+        port: 80
+        weight: 50
+```
+
+Reference the [forwardTo][forwardto] API documentation for additional details of `weight` and other fields.
+
+## Status
+
+Status defines the observed state of HTTPRoute.
+
+### RouteStatus
+
+RouteStatus defines the observed state that is required across all route types.
+
+#### Gateways
+
+Gateways define a list of the Gateways that are associated with the HTTPRoute, and the status of the HTTPRoute with
+respect to each of these Gateways. When a Gateway selects this HTTPRoute, the controller that manages the Gateway should
+add an entry to this list when the controller first sees the route and should update the entry as appropriate when the
+route is modified.
+
+The following example indicates HTTPRoute "http-example" has been admitted by Gateway "gw-example" in namespace
+"gw-example-ns":
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: http-example
+...
+status:
+  gateways:
+  - gatewayRef:
+      name: gw-example
+      namespace: gw-example-ns
+    conditions:
+    - type: Admitted
+      status: "True"
+```
+
+A maximum of 100 Gateways can be represented in this list. If this list is full, there may be additional Gateways using
+this Route that are not included in the list.
+
+[httproute]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.HTTPRoute
+[gateways]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.RouteGateways
+[httprouterule]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.HTTPRouteRule
+[hostname]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.Hostname
+[tls-config]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.RouteTLSConfig
+[rfc-3986]: https://tools.ietf.org/html/rfc3986
+[matches]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.HTTPRouteMatch
+[filters]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.HTTPRouteFilter
+[forwardto]: https://kubernetes-sigs.github.io/service-apis/spec/#networking.x-k8s.io/v1alpha1.HTTPRouteForwardTo
```

Comment:
I actually thought this was going to link to the ForwardTo section of this doc. I think it's probably better to link to the API spec like you're doing here, just not what I expected. Maybe we should make these links `target=_blank` since they're going to the spec?

Reply:
> Maybe we should make these links target=_blank since they're going to the spec?

I'm not following what you mean by `target=_blank`. Can you provide an example?

Reply:
Not important and maybe not even a good idea, would just default the link to open in a new tab/window depending on the browser.